### DISCUSSION TITLE
[5.8.1] a lot of WARN log messages for websocket clients: "emqx_session_mem_unknown_message" check_gc

### DISCUSSION THREAD

**Discussion by gbrehmer** [emqx/emqx#14197] [Q&A] [answered]
We updated today from 5.0.26 to 5.8.1 (clean installation).
Now our logfile is full with new warn level messages and I can not find any error description

One example:
```
2024-11-08T21:02:49.626424+00:00 [warning] clientid: <randomclientid>, msg: emqx_session_mem_unknown_message, peername: <ip>:443, username: <username>, message: {check_gc,#{cnt  ││ => 101,oct => 6612}}
```

All messages are similar, just another client, counts...
Is there a limit raised? Should we increase a setting somewhere? How can we disable the logging when its a false positive warning?

**Comment by zmstone** [comment, marked as answer]
Hi @gbrehmer 
Thank you for reporting it. 
I have created an issue for this. https://github.com/emqx/emqx/issues/14198
